docs/getting-started/provider-azure.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Azure Quickstart 
+weight: 3
+---
+
+Connect Crossplane to Microsoft Azure to create and manage cloud resources from Kubernetes with the [Upbound Azure Provider](https://marketplace.upbound.io/providers/upbound/provider-azure).
+
+This guide walks you through the steps required to get started with the Upbound Azure Provider. This includes installing Crossplane, configuring the provider to authenticate to Azure and creating a _Managed Resource_ in Azure directly from your Kubernetes cluster.
+
+- [Prerequisites](#prerequisites)
+  - [Install the Azure provider](#install-the-azure-provider)
+  - [Create a Kubernetes secret for Azure](#create-a-kubernetes-secret-for-azure)
+    - [Install the Azure command-line](#install-the-azure-command-line)
+    - [Create an Azure service principal](#create-an-azure-service-principal)
+    - [Create a Kubernetes secret with the Azure credentials](#create-a-kubernetes-secret-with-the-azure-credentials)
+  - [Create a ProviderConfig](#create-a-providerconfig)
+  - [Create a managed resource](#create-a-managed-resource)
+  - [Delete the managed resource](#delete-the-managed-resource)
+  - [Next steps](#next-steps)
+
+## Prerequisites
+This quickstart requires:
+* a Kubernetes cluster with at least 3 GB of RAM
+* permissions to create pods and secrets in the Kubernetes cluster
+* [Helm] version `v3.2.0` or later
+* an Azure account with permissions to create an Azure [service principal](https://learn.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals#service-principal-object) and an [Azure Resource Group](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal)
+
+{{< hint type="tip" >}}
+If you don't have a Kubernetes cluster create one locally with [minikube](https://minikube.sigs.k8s.io/docs/start/) or [kind](https://kind.sigs.k8s.io/).
+{{< /hint >}}
+
+
+{{< hint type="note" >}}
+All commands use the current `kubeconfig` context and configuration. 
+{{< /hint >}}
+
+{{< include file="/docs/master/getting-started/install-crossplane.md" type="page" >}}
+
+### Install the Azure provider
```

Comment:
This shouldn't really be a subheading of prerequisites.

Reply:
Fixed.